maat-court-data-api/src/main/java/gov/uk/courtdata/entity/ContributionFilesEntity.java

```diff
@@ -0,0 +1,47 @@
+package gov.uk.courtdata.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "CONTRIBUTION_FILES", schema = "TOGDATA")
+public class ContributionFilesEntity {
+
+    @Id
+    @SequenceGenerator(name = "contribution_files_gen_seq", sequenceName = "S_GENERAL_SEQUENCE", allocationSize = 1, schema = "TOGDATA")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "contribution_files_gen_seq")
+    @Column(name = "ID", nullable = false)
+    private Integer id;
+
+    @Column(name = "FILE_NAME", length = 45,  nullable = false)
+    private String upliftApplied;
+
+    @Column(name = "DATE_CREATED",  nullable = false)
+    private LocalDate dateCreated;
+
+    @Column(name = "USER_CREATED", length = 225, nullable = false)
```

Comment:
I think the max length for USER_CREATED should be 250 as per the DB schema

> 	"USER_CREATED" VARCHAR2(250 BYTE) DEFAULT user NOT NULL ENABLE,

Reply:
Most of the fields from this entity will be removed.